taskcluster/ci/github-release/kind.yml

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: ac_taskgraph.loader.all_dep:loader
+
+transforms:
+    - ac_taskgraph.transforms.github_release:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - post-beetmover
+
+only-for-build-types:
+    - beta
```

Comment:
Good call on not including `nightly` here. We don't want nighlies to be included on Github releases. That will be too noisy, in my opinion. 

That said, we don't have A-C betas. So we can remove anything related to `beta` there. If one day we end up needing betas, then we can revisit this later.